src/utils/runPkgManagerInstall.ts

```diff
@@ -34,11 +34,13 @@ export const runPkgManagerInstall = async (
         continue;
       }
 
+      const pkgName = pkg.replace(/^(@?[^@]+)(?:@.+)?$/, "$1");
```

Comment:
Not good with regex, would this support scoped packages too?

For example if we need to lock tRPC to a version like `@trpc/react@9.26.2`?

Reply:
yup, it works with scoped package.

here is some examples
![image](https://user-images.githubusercontent.com/94958349/179705979-3a06d4eb-fe44-4904-9934-baeeecbeba9b.png)